### DISCUSSION TITLE
Multiplayer mod

### DISCUSSION THREAD

**Discussion by GTAGAMERVI** [luckydog7/Funkin-android#1431] [Q&A]
ah help me

**Comment by NeptuneNeptix** [comment]
nope.

**Reply by zsidebf** [reply]
Why

**Reply by whateverlior** [reply]
> nope.

Who asked you?

**Reply by NeptuneNeptix** [reply]
> > nope.
> 
> Who asked you?

ok and?

**Reply by NEWLuigi64** [reply]
> > > nope.
> > 
> > 
> > Who asked you?
> 
> ok and?

It's a simple question, *WHO ASKED*
